tests/test_cookie_signing.py

```diff
@@ -64,6 +64,13 @@ def test_set_and_get_cookie_with_existing_cookie():
     assert REQUEST.get_cookie('test') == 'testvalue'
 
 
-def test_set_and_get_cookie_http_only():
-    REQUEST.cookie('test', 'testvalue', http_only=True)
+def test_set_and_get_cookie_with_http_only():
+    REQUEST.cookie('test', 'testvalue')
+    assert REQUEST.get_cookie('test') == 'testvalue'
+    assert 'HttpOnly' in REQUEST.cookies[0][1]
+
+
+def test_set_and_get_cookie_without_http_only():
```

Comment:
Ok so the above test is good but the test below is bad. The test below should be something like:

```python
REQUEST.cookies = [] # reset the cookie list
REQUEST.cookie('test', 'testvalue', http_only=False)
assert 'test=testvalue; Path=/' in REQUEST.cookies[0][1]
```

I'm not sure the test below shouldn't have passed because it implies that the string is empty.

So just for your information:

REQUEST.cookies[0] should return `('Set-Cookie', 'key=value; HttpOnly ... blah blah')`

then REQUEST.cookies[0][1] should return `'key=value; HttpOnly ... blah blah'`

make sure the cookie is constructed correctly